<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_dialog</class>
 <widget class="QWidget" name="register_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>560</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>register_dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>510</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>允许操作</string>
   </property>
   <widget class="QCheckBox" name="check_call">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>打电话</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_presence">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>呈现</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>510</width>
     <height>310</height>
    </rect>
   </property>
   <property name="title">
    <string>用户设置</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>10</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_account">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>账户</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_account">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>34</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>10</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_domain">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>域名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_domain">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_passwd">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>145</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_passwd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_display_name">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>205</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_authorize">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>270</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>鉴权名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_authorize">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color:rgb(233, 233, 233);
    background-repeat:no-repeat;
    border:none;
}
QPushButton:hover{
    background-color:#CCCCCC
}
QPushButton:pressed{
    background-color:#888888
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color:rgb(233, 233, 233);
    background-repeat:no-repeat;
    border:none;
}
QPushButton:hover{
    background-color:#CCCCCC
}
QPushButton:pressed{
    background-color:#888888
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_account_top">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>账户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_protol">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>协议</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_account_top">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Account 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_protol_top">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(172, 172, 172);</string>
   </property>
   <property name="text">
    <string>SIP</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
